test/core/runtime/wasm_result_test.cpp

```diff
@@ -33,7 +33,7 @@ TEST_P(WasmResultTest, DecomposeSuccess) {
   auto [res, pair] = GetParam();
   auto [address, length] = pair;
 
-  WasmResult r{res};
+  WasmResult r(res);
```

Comment:
It the same what was =)

Reply:
Some compilers won't agree with you.

Reply:
Let's trying? =)
Answered [above](https://github.com/soramitsu/kagome/pull/353#discussion_r444091987)